crates/bevy_math/src/gcd.rs

```diff
@@ -0,0 +1,168 @@
+//! Provides implementations for methods relating to the [Greatest Common Divisor](https://en.wikipedia.org/wiki/Greatest_common_divisor).
+
+/// Computes the [Greatest Common Divisor](https://en.wikipedia.org/wiki/Greatest_common_divisor)
+/// between the supplied `value` and the constant parameter `N`.
```

Comment:
Since this function has parameters `a` and `b` instead of `value` and a constant `N`, the comment should be changed accordingly.